apps/web/src/components/safe-messages/MsgDetails/index.tsx

```diff
@@ -46,7 +46,7 @@ const MsgDetails = ({ msg }: { msg: SafeMessage }): ReactElement => {
             <EthHashInfo
               address={msg.proposedBy.value || ''}
               name={msg.proposedBy.name}
-              customAvatar={msg.proposedBy.logoUri}
+              customAvatar={msg.proposedBy.logoUri || undefined}
```

Comment:
Afaiu logoUri now has the type of `string | null | undefined`. Is there a benefit of having a fallback here instead of adjusting the `EthHashInfo` type to also accept `null` as a value for `customAvatar`?

Reply:
Although thinking about it it makes sense to change it here instead of everywhere downstream so we don't have to deal with a null value in potentially many components.

Reply:
Good question. The EthHashInfo component has specific requirements. if you pass a prop to it, it should be a string, otherwise you should not pass a customAvatar. The component defines concrete function arguments. 
I'm not sure changing the function arguments should be done, because an API we use might return an incompatible type. 
On the other hand I agree that it is a bit weird, but not sure that deserves changing the EthHashInfo component.